tests/wpt/metadata-css/css-backgrounds-3_dev/html4/attachment-local-clipping-image-6.htm.ini

```diff
@@ -0,0 +1,3 @@
+[attachment-local-clipping-image-6.htm]
```

Comment:
Here is the issue which I think is causing the failure in this test: https://github.com/servo/servo/issues/17718